<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlendDialog</class>
 <widget class="QDialog" name="BlendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interpolation options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Interpolation</string>
   </property>
   <widget class="QLabel" name="From">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;From&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="To">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;To&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Animation1">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Animation 1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Animation2">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Animation 2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Frame1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Frame2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_frameCount0">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>68</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_frameCount1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>118</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Description">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose range of frames for blending:</string>
    </property>
   </widget>
   <widget class="QLabel" name="For">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>45</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;For&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="Using">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>frames, using</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_cubicRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cubic</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_slerpRadioButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SLERP</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m_blendRange">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>168</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_totFrame0">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>70</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>120</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_totFrame1">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>120</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BlendDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BlendDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
